src/analysis/complex/locally_uniform_limit.lean

```diff
@@ -0,0 +1,167 @@
+/-
+Copyright (c) 2022 Vincent Beffara. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Vincent Beffara
+-/
+import analysis.complex.removable_singularity
+import analysis.calculus.uniform_limits_deriv
+
+/-!
+# Locally uniform limits of holomorphic functions
```

Comment:
Could you mention somewhere that this is for functions defined on `ℂ`? (The same result is true for a general normed vector space over `ℂ`, but harder). Also, can you sketch the proof you use in this file-level docstring?